spk/salt-master/src/requirements.txt

```diff
@@ -1,13 +1,11 @@
-salt==2015.5.11
-Jinja2==2.11.2
-requests==2.23.0
-CherryPy==11.2.0
-more-itertools==5.0.0
-cheroot==6.3.3
-backports.functools-lru-cache==1.4
-jaraco.windows==3.7
-jaraco.classes==1.4.3
-jaraco.functools==1.17
-portend==2.2
-tempora==1.10
-futures==3.3.0
+# https://github.com/saltstack/salt/blob/master/requirements/static/pkg/py3.8/linux.txt
+certifi==2021.5.30
```

Comment:
This file should now be renamed to `requirements-pure.txt`.
Also, certifi is one of the handful of wheels provided with python310.